src/pages/minjae/Login/Login.js

```diff
@@ -1,13 +1,107 @@
-import React, { Component } from 'react'
+import React, { Component } from "react";
+import "./Login.scss";
+import "../Nav/Nav";
+import "../../../styles/common.scss";
 
 class Login extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      disabled: true,
+      btnOpacity: "30%",
+      iptIdValue: "",
+      iptPwValue: "",
+      phone: "01048392736",
+      nickname: "pie",
+    };
+  }
+
+  handleIdInput = (e) => {
+    this.setState({ iptIdValue: e.target.value });
+  };
+
+  handlePwInput = (e) => {
+    this.setState({ iptPwValue: e.target.value });
+  };
```

Comment:
```suggestion
  handleInputs = e => {
    const { name, value } = e.target;
    
    this.setState({ [name]: value });
  };
```
이런식으로 하나의 함수로 합칠 수 있겠네요!
그렇다면 input 태그에 각각 name="iptIdValue" 이런식으로 부여를 해주셔야합니다!